docs/app/pages/components/components-sections/tabs/tabs/tabs.component.html

```diff
@@ -13,9 +13,8 @@ <h4>{{ tab.title }}</h4>
             </div>
         </ux-tab>
     </ux-tabset>
-
+    <button class="m-b btn button-secondary" (click)="tabset.selectTab(1)">Select Second Tab</button>
```

Comment:
You can use either use bootstrap margin classes (`m-b`) or custom styles, but I would prefer that you don't use both within the same example.